infra/nightly-resources/chart.js

```diff
@@ -81,7 +84,7 @@ function parseDataForChart() {
     datasets[mode] = {
       label: mode,
       backgroundColor: COLORS[mode],
-      data: Array(benchmarks.length).fill(0),
+      data: Array(benchmarks.size).fill(0),
```

Comment:
did you mean benchmarks.size here? Idk the difference but a cursory google search says length is right for arrays

Reply:
Yeah, it's a set not an array